### DISCUSSION TITLE
How to reply to message

### DISCUSSION THREAD

**Discussion by MageTea** [Chatterino/chatterino2#4184] [Q&A] [answered]
I don't see the option to reply to a message when I right click and I can't find anything in settings related to replies. Is there something I am missing that needs to be enabled?

**Comment by Mm2PL** [comment, marked as answer]
Have you tried updating to (newest) nightly or 2.4.0-beta

**Reply by MageTea** [reply]
Oh! Thank you. I had overlooked that it was a new feature. I was using 2.3.5.
